charts/platform-reports/values.yaml

```diff
@@ -28,6 +28,38 @@ imagePullSecrets: []
 
 secrets: []
 
+additionalScrapeConfigs:
+  - job_name: nvidia-dcgm-exporter
+    honor_labels: true
+    relabel_configs:
+    - sourceLabels: [__name__]
+      targetLabel: np-cluster-name
+      replacement: placeholder
```

Comment:
I am not sure I follow what we are trying to do here.
please elaborate on the logic behind these rules.

Reply:
The logic here was as follows:
1. Take each scrape job we have
2. Add a new metric label for each metric by replacing previous ones (preserving them) with the placeholder value.

I didn't really like this approach, since:
- it would (probably) create more load on Prometheus server (since we take each label in 2)
- I wasn't sure re scrape job names

Therefore, I've replaced this approach with a different one in [23bf3f3](https://github.com/neuro-inc/platform-reports/pull/626/commits/23bf3f3c86d9ea64471b2883bda6fd22f6fbeb05): we add extra labels to all metrics when we communicate them from Prometheus (in our case, to Thanos).

For this approach to work, we should specify `kube-prometheus-stack.prometheus.prometheusSpec.externalLabels.np-cluster-name` value for this chart on it's deployment.
I don't really like this neither, since we need to patch platform-operator too, but cannot find a more elegant way to template this or copy value from `platform.clusterName` (Prometheus CRD uses this config [as-is](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml#L56) w.o. allowing to use some external config-map or secret as with additionalScrapeConfigsSecret)

Reply:
wdyt on this @dalazx , @zubenkoivan

Reply:
@YevheniiSemendiak Could you check that label is present for fresh metrics and old metrics (> 24h)?

Reply:
Updating only `values.yaml` is enough, you don't need to update `values-dev.yaml` and platform-operator

Reply:
@zubenkoivan but how we the value be set without patching `platform-operator`?

Reply:
@zubenkoivan - I've added test to verify label is present for fresh metrics.
Re old metrics, if you meant those which we already have in our existing cluster - those will be absent. Thanos will create new time series for **all metrics** since our previous ones did not have a `cluster` label in them. But for the new ones the label will be available, yes.

Reply:
@dalazx values in `values.yaml` are considered default values by helm, they are published to registry and always applied unless you override them

Reply:
@zubenkoivan so where are we going to override it?

Reply:
will do that as part of this https://github.com/neuro-inc/platform-operator/issues/606